src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/AppealService.kt

```diff
@@ -92,9 +92,14 @@ class AppealService(
 
         saveEvent(appeal)
 
-        if (decision == AppealDecision.accepted) {
-          assessmentService.createApprovedPremisesAssessment(application as ApprovedPremisesApplicationEntity, createdFromAppeal = true)
-          cas1AppealEmailService.appealSuccess(application, appeal)
+        when (decision) {
```

Comment:
'when' is 50% of reason why i'd use kotlin over java going forward, makes everything so much safer!

Reply:
Yeah, it's ace!